### ISSUE TITLE
USER ISSUE: fishing pole does not work

### ISSUE THREAD

**Issue by pfmd86** [StrangeLoopGames/EcoIssues#18349]
\
**Version:** 0.9.0.2 beta release-71

**Steps to Reproduce:**
using the fishing pole

**Expected behavior:**
throwing thread in front of me

**Actual behavior:**
thread is [visible](url
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/5250428/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/5250429/Player-prev.log)
![2020-09-19 11_35_36-Greenshot](https://user-images.githubusercontent.com/8144485/93686637-5bb19c80-fa6c-11ea-89dd-e169ec8ba108.png)

) in the top right corner, water noise playing non-stop

**Do you have mods installed? Does the issue happen when no mods are installed?:**
no mods

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@pfmd86 Thank you for the report.
1. Can you provide video?
2. Please, provide graphics setting(screenshot).
3. Provide your PC specification.

**Comment by pfmd86** [comment]
Here's the graphics settings:
![2020-09-29 13_46_07-Greenshot](https://user-images.githubusercontent.com/8144485/94613957-41d93c00-025a-11eb-9c13-b014dc1a1008.png)

Here the Video:
https://www.youtube.com/watch?v=AN2ECwxdkYs

You can see the pole in the top right corner of the screen. I tried several resolutions from 720 to 4K, all the same.

PC:
Alienware 17 R5
Windows 10 Home
32GB RAM
NVIDIA GeForce GTX 1080

**Comment by beastyn** [comment]
It seems we found it.
@MazonDel you need a second client for this test.
1) Player1 and Player 2 stay together
2) Player1 cast a fishing pole
3) Player 2 fly away and unload a chank with Player1.
4) Player 1 got an error and pole line is broken

```
NullReferenceException: Object reference not set to an instance of an object.
  at AkAmbientLargeModePositioner.get_Position () [0x00000] in <00000000000000000000000000000000>:0 
  at FishingController.GetPoints () [0x00000] in <00000000000000000000000000000000>:0 
  at FishingController.Update () [0x00000] in <00000000000000000000000000000000>:0 
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
 
(Filename: currently not available on il2cpp Line: -1)```

**Comment by pfmd86** [comment]
Is that fixed in the current public version? My Server and Client are up-to-date but I still have the same problem.

**Comment by IcedForge** [comment]
@pfmd86 No this is fixed for 9.1
